src/animation.jl

```diff
@@ -241,5 +284,5 @@ gif(anim)
 ```
 """
 macro animate(forloop::Expr, args...)
-    _animate(forloop, args...)
+    _animate(forloop, args...; type = :none)
```

Comment:
```suggestion
    _animate(forloop, args...)
```

No need to specify the `kwarg` since `type` defaults to `:none`.